INSTALL.md

```diff
@@ -2,24 +2,25 @@
 
 Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
 
-1.) Using helm commands: <br />
+1.) Installation from released chart: <br />
 
 How to install application using helm:  <br />
     helm install ReleaseName ChartName
     
-    a.) Add helm repository in tractusx:
-           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/dev
+    a.) Add helm repository for sdfactory:
```

Comment:
It's not the helm repository for sd factory, it's helm chart repository for tractusx-dev

Reply:
Taken care